### PR TITLE
add drone-s3 secrets to documentation

### PR DESCRIPTION
**Pull request by Ashtonian**
part of https://github.com/drone-plugins/drone-s3/issues/48

### CODE REVIEW

File: content/drone-plugins/drone-s3/index.md [hunk 1]
```diff
@@ -72,6 +86,14 @@ pipeline:
 +   endpoint: https://play.minio.io:9000
 ```
 
+# Secret Reference
+
+aws_access_key_id
```
Comment (tboerger) [inline content/drone-plugins/drone-s3/index.md]: Could you add the other possible variable names as well, please? Same for the other secret. It should be just a comma separated string.

Comment (Ashtonian) [reply content/drone-plugins/drone-s3/index.md]: like that ?

Comment (tboerger) [reply content/drone-plugins/drone-s3/index.md]: Just checked the source, sorry... Your first commit was right. 

But there are other secrets available like s3_region, s3_endpoint and s3_bucket.

**Comment by d1wilko** [comment]
Hi,

We have recently updated the drone plugin index and docs are no longer written in markdown - they are now a more consistent yaml format e.g. [docker](https://github.com/drone/drone-plugin-index/blob/master/plugins/docker/content.yaml)

Please have a look at the new readme if you would like to run the new plugin index [readme](https://github.com/drone/drone-plugin-index/blob/master/README.md)

If you would still like to add documentation to the plugins index can you change the format to
- An individual folder per plugin inside the `plugins` folder
- a `content.yaml` file that follows the format in the example above - this is for quick discovery, not full documentation
- an `original.md` file that can contain the markdown you have already written
- a logo image inside the `public/logos` folder

Thanks :)

**Pull request closed without merging**